pkg/activator/net/helpers.go

```diff
@@ -49,7 +49,7 @@ func endpointsToDests(endpoints *corev1.Endpoints, portName string) sets.String
 // getServicePort takes a service and a protocol and returns the port number of
 // the port named for that protocol. If the port is not found then ok is false.
 func getServicePort(protocol networking.ProtocolType, svc *corev1.Service) (port int, ok bool) {
-	wantName := networking.ServicePortName(protocol)
+	wantName := networking.ServicePortName(protocol) + "-proxy"
```

Comment:
This should probably be hidden inside `networking` package.

Reply:
Sure, in the proper impl. we'll have a "helper" for this or something.